src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -340,28 +316,14 @@ export const VenueLandingPage: React.FC = () => {
                             </div>
                           )}
 
-                          {user ? (
-                            <EventPaymentButton
-                              event={venueEvent}
-                              venueId={venueId}
-                              isUserVenueOwner={!!isUserVenueOwner}
-                              selectEvent={() => setSelectedEvent(venueEvent)}
-                              setIsPaymentModalOpen={setIsPaymentModalOpen}
-                              paymentConfirmationPending={
-                                eventPaidSuccessfully === venueEvent.id
-                              }
-                            />
-                          ) : (
-                            <button
-                              className="btn btn-primary buy-tickets-button"
-                              onClick={() => {
-                                setSelectedEvent(venueEvent);
-                                openAuthenticationModal();
-                              }}
-                            >
-                              Buy tickets
-                            </button>
-                          )}
+                          <button
+                            className="btn btn-primary buy-tickets-button"
+                            onClick={() => {
+                              openAuthenticationModal();
+                            }}
+                          >
+                            Buy tickets
```

Comment:
We're removing payments, so there will be no way to buy tickets. So this whole button, or potentially even this whole section will likely want/need to be removed? You'd need to look more into the logic of this page to make that decision properly.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40